org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java

```diff
@@ -42,4 +41,30 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOExce
 		Collections.sort(matchedList, String.CASE_INSENSITIVE_ORDER);
 		return matchedList;
 	}
+
+	public static String readFile(String filename) {
+		BufferedReader br = null;
+		StringBuilder sb = new StringBuilder();
+		try {
+
+			String sCurrentLine;
+
+			br = new BufferedReader(new FileReader(filename));
+
+			while ((sCurrentLine = br.readLine()) != null) {
+				sb.append(sCurrentLine);
+			}
+
+		} catch (IOException e) {
+			e.printStackTrace();
+		} finally {
+			try {
+				if (br != null)
+					br.close();
```

Comment:
There's a high change that this method won't be necessary anymore, once you perform the changes suggested above.
Nevertheless: Closing BufferedReader (and other `Closeable` implementations) is not necessary anymore.
Instead, you can use the [try-with-resources][1] construct.

[1]: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html